ios/RNSentry.m

```diff
@@ -171,33 +171,27 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
     NSLog(@"Bridge call to: deviceContexts");
-    NSMutableDictionary<NSString *, id> *contexts = [NSMutableDictionary new];
+    __block NSMutableDictionary *contexts;
     // Temp work around until sorted out this API in sentry-cocoa.
     // TODO: If the callback isnt' executed the promise wouldn't be resolved.
     [SentrySDK configureScope:^(SentryScope * _Nonnull scope) {
-        NSDictionary<NSString *, id> *serializedScope = [scope serialize];
-        // Scope serializes as 'context' instead of 'contexts' as it does for the event.
-        NSDictionary<NSString *, id> *tempContexts = [serializedScope valueForKey:@"context"];
-
-        NSMutableDictionary<NSString *, id> *user = [NSMutableDictionary new];
-
-        NSDictionary<NSString *, id> *tempUser = [serializedScope valueForKey:@"user"];
-        if (tempUser != nil) {
-            [user addEntriesFromDictionary:[tempUser valueForKey:@"user"]];
-        } else {
-            [user setValue:PrivateSentrySDKOnly.installationID forKey:@"id"];
+        NSDictionary *serializedScope = [scope serialize];
```

Comment:
Is there a reason to avoid the specific types such as `<NSString *, id>`?

Reply:
Sind I didn't need the specific type it was simple to read for me. But other than that no reason. I will return it.

Reply:
I'd rather prefer strongly typed, so the compiler helps us if we do something wrong.